.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         pip install -U pip
 
         curl -sSL \
-          "https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py" | python
+          "https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py" | python -
```

Comment:
Let's add also `poetry` to `${PATH}`:

```bash
# add `poetry` to `${PATH}`
echo "${HOME}/.poetry/bin" >> ${GITHUB_PATH}
```

References:

+ https://python-poetry.org/docs/#osx-linux-bashonwindows-install-instructions
+ https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

Reply:
Thanks, it should be fine now I think